sqlglot/tokens.py

```diff
@@ -226,6 +226,7 @@ class TokenType(AutoName):
     ARRAY = auto()
     ASC = auto()
     ASOF = auto()
+    ATTACH = auto()
```

Comment:
these tokens need to be added to identity so you can do 'select 1 attach' right?

Reply:
Right, will add the relevant identity tests. Do note that DuckDB doesn't allow them as valid identifiers, so since they're only defined in `duckdb.py` I thought the current solution suffices:

```SQL
D select 1 attach;
Parser Error: syntax error at or near "attach"
LINE 1: select 1 attach;
                 ^
D select 1 detach;
Parser Error: syntax error at or near "detach"
LINE 1: select 1 detach;

D select 1 a;
┌───────┐
│   a   │
│ int32 │
├───────┤
│     1 │
└───────┘
```